### DISCUSSION TITLE
Unsteady LES compressible simulations for Aeroacoustics cases

### DISCUSSION THREAD

**Discussion by Omjagtap1** [su2code/SU2#1296] [General]
Hey,

I am new to SU2, wanted to know if I can do unsteady LES compressible simulation for aeroacoustics cases. I saw the tutorial case of Unsteady NACA0012 with RANS. Curious to know if I can modify and use it for my case.

**Comment by Omjagtap1** [comment]
Can someone give some idea about this?! Any help would be great!

**Reply by TobiKattmann** [reply]
I am not sure what the current status of LES is in SU2 as there is an open Pull Request on this #1120. The author is @EduardoMolina  so he probably knows best.
